### PR TITLE
Add German localization

### PR DESCRIPTION
**Pull request by ghost**
Also fix minor mistakes in the English strings where encountered

### CODE REVIEW

**Comment by ghost** [comment]
I have a squashed lined up in another branch, see ya over there. #388

File: masterlist.yaml [hunk 1]
```diff
@@ -70,12 +80,16 @@ common:
         text: 'SkyUI_SE.esp is not active in your load order. [SkyUI SE](https://www.nexusmods.com/skyrimspecialedition/mods/12604) is required for this mod to be fully functional.'
       - lang: ja
         text: 'SkyUI_SE.espがアクティブになっていません。このModを完全に機能させるには[SkyUI SE](https://www.nexusmods.com/skyrimspecialedition/mods/12604)が必要です。'
+      - lang: de
+        text: 'SkyUI_SE.esp ist nicht aktiv in Ihrer Ladereihenfolge. [SkyUI SE](https://www.nexusmods.com/skyrimspecialedition/mods/12604) is required for this mod to be fully functional.'
```
Comment (MacSplody) [inline masterlist.yaml:84]: > is required for this mod to be fully functional.

Comment (ghost) [reply masterlist.yaml:84]: ofc I had to miss something

**Comment by MacSplody** [comment]
> I have a squashed lined up in another branch, see ya over there. #388

For future reference, I can squash when merging, so it's not really required to close & open a new pull.

**Comment by ghost** [comment]
Understood.

**Comment by MacSplody** [comment]
No worries, thanks for the contribution.

**Pull request closed without merging**